### ISSUE TITLE
unused variable, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1214]
```abap
DATA r_result TYPE string.
SELECT column INTO @r_result UP TO 1 ROWS FROM voided_table.
ENDSELECT.
```
